processing/src/main/java/org/apache/druid/query/context/ResponseContext.java

```diff
@@ -319,7 +323,11 @@ public void merge(ResponseContext responseContext)
 
   /**
    * Serializes the context given that the resulting string length is less than the provided limit.
-   * This method tries to remove some elements from context collections if it's needed to satisfy the limit.
+   * This method removes some elements from context collections if it's needed to satisfy the limit.
+   * There is no explicit priorities of keys which values are being truncated because for now there are only
+   * two potential limit breaking keys (UNCOVERED_INTERVALS and MISSING_SEGMENTS) and their values are arrays.
```

Comment:
Please wrap `UNCOVERED_INTERVALS` and `MISSING_SEGMENTS` with `{@link }`

Reply:
wrapped